charts/k8spacket/Chart.yaml

```diff
@@ -5,5 +5,5 @@ description: A Helm chart for k8spacket tool
 maintainers:
   - name: k8spacket
     email: k8spacket@gmail.com
-version: 2.0.5
-appVersion: 2.0.5
\ No newline at end of file
+version: 2.0.6
+appVersion: 2.0.6
```

Comment:
`appVersion` should stay as `2.0.5` because it is the application version (= default docker version if `image.tag` is not explicitly set. See https://github.com/k8spacket/k8spacket-helm-chart/blob/master/charts/k8spacket/templates/daemonset.yaml#L60)

Reply:
Thanks for the help and observations @k8spacket. I made the correction.